rq/serializers.py

```diff
@@ -1,21 +1,22 @@
 import json
 import pickle
 from functools import partial
-from typing import TYPE_CHECKING, Callable, Optional, Type, Union
+from typing import Any, Callable, ClassVar, Optional, Protocol, Union, runtime_checkable
 
 from .utils import import_attribute
 
-if TYPE_CHECKING:
-    from typing_extensions import Protocol
 
-    class Serializer(Protocol):
-        dumps: Callable[..., bytes]
-        loads: Callable[..., object]
+@runtime_checkable
+class Serializer(Protocol):
+    def dumps(self, obj: Any, /) -> bytes:
+        ...
```

Comment:
Mind adding ` # pragma: no cover` at the end of the `...` lines so Codecov doesn't mark is as untested?

Reply:
Sure; done.